### PR TITLE
Setup Renovate to update third-party parser tests

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
| Q                        | A 
| ------------------------ | ---
| License                  | MIT

This is based on https://github.com/renovatebot/renovate/discussions/10381#discussioncomment-855411 (thanks @rarkins!). If it works, we can avoid maintaining the custom action to udpate test262 tests.

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/13454"></a>

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,15 +1,50 @@
...
+      "fileMatch": ["^Makefile$"],
+      "matchStrings": ["TYPESCRIPT_COMMIT = (?<currentDigest>.*)\\n"],
+      "datasourceTemplate": "git-refs",
+      "depNameTemplate": "tests-typescript",
+      "lookupNameTemplate": "https://github.com/Microsoft/TypeScript.git",
+      "currentValueTemplate": "main"
+    },
+    {
+      "fileMatch": ["^Makefile$"],
+      "matchStrings": ["COMPAT_TABLE_COMMIT=(?<currentDigest>.*)\\n"],
```
Comment (nicolo-ribaudo) [inline renovate.json:33]: This isn't automatically updated by this PR, but it should appear in the dependencies dashboard.

**Comment by rarkins** [comment]
@nicolo-ribaudo sorry, I just realized that I needed to make a small change to regex manager when I was testing locally. I will commit it and push it to the app and let you know when it's ready for this PR to be merged.

**Comment by nicolo-ribaudo** [comment]
Thanks you!

**Comment by rarkins** [comment]
Ready :)

File: renovate.json [hunk 2]
```diff
@@ -1,15 +1,50 @@
...
+    }
+  ],
   "packageRules": [
     {
       "matchDepTypes": ["dependencies", "devDependencies"],
       "rangeStrategy": "replace",
-      "ignoreDeps": ["@babel/core-7.12", "@babel/helper-validator-identifier-baseline"]
+      "ignoreDeps": [
+        "@babel/core-7.12",
+        "@babel/helper-validator-identifier-baseline"
```
Comment (JLHwung) [inline renovate.json:46]: I can rename the package so we don't have to ignore it here.

Comment (JLHwung) [inline renovate.json:46]:
```suggestion
        "@babel/helper-validator-identifier-baseline",
        "@babel-baseline/parser",
```

**Review by JLHwung — APPROVED** [review]

**Review by existentialism — APPROVED** [review]

**Pull request merged**